scripts/clear-all-faults.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Wondering if we can make this /bin/sh vs. /bin/bash.

Reply:
There is no particular reason for selecting bash. in case we add more features to the same script may require bash in future